pkg/morph/client/container/client.go

```diff
@@ -89,9 +89,15 @@ type opts struct {
 }
 
 func defaultOpts() *opts {
-	var o = new(opts)
-	o.staticOpts = append(o.staticOpts, client.TryNotary())
-	return o
+	return new(opts)
```

Comment:
Seems too dangerous to me. Can we have `DisableNotary()` specifically to use in those cases where its' needed?

Reply:
And to be fair it's not about enabling/disabling notary, it's more about using appropriate signers. This `TryNotary()` name always looked wrong to me.

Reply:
So we still change default?

Reply:
i pushed smth strange, check one more time, please